### PR TITLE
Removed unused system settings for "File System" section

### PR DESCRIPTION
**Pull request by Ruslan-Aleev**
### What does it do?
Removed unused system settings for "File System" section.

These settings are **outdated** or found only in the "System Settings", do not participate in the remaining code:
- `filemanager_path`
- `filemanager_path_relative`
- `filemanager_url`
- `filemanager_url_relative`
- `rb_base_dir`
- `rb_base_url`
- `strip_image_paths`
- `use_browser`

I'm not quite sure about the my corrections in the code (https://github.com/modxcms/revolution/pull/14886/commits/f62cac8f95f10dc032ee0741b997e64b81a50132), maybe there is a more competent solution.

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/13458
https://github.com/modxcms/revolution/issues/14539#issuecomment-482059233

### CODE REVIEW

File: manager/assets/modext/widgets/element/modx.panel.tv.renders.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ Ext.reg('modx-panel-tv-image',MODx.panel.ImageTV);
 
 MODx.panel.FileTV = function(config) {
     config = config || {};
-    config.filemanager_url = MODx.config.filemanager_url;
+    config.filemanager_url = '';
```
Comment (alroniks) [inline manager/assets/modext/widgets/element/modx.panel.tv.renders.js]: Does it make sense to find usage of this config param and remove it too?

File: manager/assets/modext/widgets/element/modx.panel.tv.renders.js [hunk 2]
```diff
@@ -8,7 +8,7 @@
  */
 MODx.panel.ImageTV = function(config) {
     config = config || {};
-    config.filemanager_url = MODx.config.filemanager_url;
+    config.filemanager_url = '';
```
Comment (alroniks) [inline manager/assets/modext/widgets/element/modx.panel.tv.renders.js]: Does it make sense to find usage of this config param and remove it too?

File: manager/assets/modext/widgets/element/modx.panel.tv.renders.js (hunk 1 above)
Comment (Ruslan-Aleev) [reply manager/assets/modext/widgets/element/modx.panel.tv.renders.js]: In my opinion, this setting is not used anywhere (the search finds only in this file), apparently it is more correct to delete `config.filemanager_url`

Comment (alroniks) [reply manager/assets/modext/widgets/element/modx.panel.tv.renders.js]: What I am talking about :)

**Comment by Ruslan-Aleev** [comment]
@Alroniks I corrected the code, please test PR

**Review by alroniks — APPROVED** [review]

**Review by JoshuaLuckers — APPROVED** [review]

**Comment by alroniks** [comment]
@Ruslan-Aleev Could you rebase changes of your branch?

**Pull request merged**